include/oneapi/tbb/detail/_flow_graph_impl.h

```diff
@@ -349,30 +346,50 @@ class graph : no_copy, public graph_proxy {
 
     graph_task_priority_queue_t my_priority_queue;
 
+    d1::wait_context_vertex& get_wait_context_vertex() { return my_wait_context_vertex; }
+
     friend void activate_graph(graph& g);
     friend void deactivate_graph(graph& g);
     friend bool is_graph_active(graph& g);
+    friend bool is_thread_in_graph_arena(graph& g);
```

Comment:
Suggestion is to rename to indicate that this is exactly this thread, not any one else.
```suggestion
    friend bool is_this_thread_in_graph_arena(graph& g);
```

Reply:
Done